### ISSUE TITLE
Hotkeys not working on arch Linux

### ISSUE THREAD

**Issue by 404GH0ST** [ILikeAI/AlwaysReddy#38]
```bash
❯ python main.py
Using faster-whisper model: base.en and device: cuda


Press 'alt+ctrl+r' to start recording, press again to stop and transcribe.
Double tap to the record hotkey to give AlwaysReddy the content currently copied in your clipboard.
Press 'alt+ctrl+e' to cancel recording.
Press 'alt+ctrl+w' to clear the chat history.
^[^R
^[^[^R
^[^R          
^[^[^R
^[^[
```
Nothing happened after I pressed the hotkeys.

**Comment by ILikeAI** [comment]
These damn hotkeys man haha. I have had a lot of issues setting up the input library to work between ubuntu and linux, someone has suggested a new input library that may fix these issues. Its a busy time for me right now but the next thing on my always reddy to do list is to try to fix these issue!
Sorry about that

**Comment by 404GH0ST** [comment]
It's okay; anyway, it's a great project. It works well on my Windows machine. I think I will migrate my Linux window manager to bspwm (X11) until there's a fix for hotkeys in Wayland.

**Comment by ILikeAI** [comment]
I looked at the alternative inputs library yesterday and unfortunately had no luck, not entirely sure what the solution will be

**Comment by 404GH0ST** [comment]
Yeah, I think there is no reliable input library for Wayland yet. In the meantime, Xorg works well, just like on a Windows machine. I can call the hotkeys on any workspace while the program is running on one workspace.

**Comment by ILikeAI** [comment]
ahhh that makes me feel a lot better to hear lol, i was starting to worry that i was just dumb!

**Comment by Theo-codatte** [comment]
Fun fact, I can setup some hotkeys with the GUI
But they are not working during AE usage
